OpenShift/helm/nexus-iq/templates/service.yaml

```diff
@@ -1,13 +1,19 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ template "iqserver.fullname" . }}
+  name: {{ include "iqserver.fullname" . }}
+  labels:
+    {{- include "iqserver.labels" . | nindent 4 }}
 spec:
-  selector:
-    app: {{ template "iqserver.fullname" . }}
+  type: {{ .Values.service.type }}
```

Comment:
I notice that the NOTEST.txt references `serviceType` but this is `service.type`. Are they supposed to be the same thing?